cmd/cephcsi.go

```diff
@@ -106,6 +106,10 @@ func main() {
 		klog.Fatalln(err) // calls exit
 	}
 	klog.Infof("Starting driver type: %v with name: %v", driverType, dname)
+	err = util.CreateKeyRingFiles()
```

Comment:
If we are moving keyring creation here, I would suggest moving calling `util.WriteCephConfig` as well to the same location. Not a must fix item, but more a cleanup as we do this.

Reply:
moved keyring creation to `WriteCephConfig` function